plugins/catalog-import/src/components/ImportStepper/ImportStepper.tsx

```diff
@@ -42,18 +43,19 @@ type Props = {
     defaults: StepperProvider,
   ) => StepperProvider;
   variant?: InfoCardVariants;
-  opts?: StepperProviderOpts;
+  /// @deprecated Pass import options via ImportOptionsContext instead.
+  opts?: ImportOptions;
 };
 
 export const ImportStepper = ({
   initialUrl,
   generateStepper = defaultGenerateStepper,
   variant,
-  opts,
 }: Props) => {
   const configApi = useApi(configApiRef);
   const classes = useStyles();
   const state = useImportState({ initialUrl });
+  const opts = useImportOptions();
```

Comment:
if we deprecate rather than remove the `opts` prop, should this code do `const opts = props.opts || useImportOptions()` instead? Ok, but not using the `||` since it's a hook